docs/scripts/formattedTSDemos.js

```diff
@@ -21,13 +21,8 @@ const { fixBabelGeneratorIssues, fixLineEndings } = require('./helpers');
 
 const tsConfig = typescriptToProptypes.loadConfig(path.resolve(__dirname, '../tsconfig.json'));
 
-const unwrap = babel.createConfigItem(
-  require('../node_modules/@material-ui/monorepo/packages/babel-plugin-unwrap-createstyles'),
-);
-
 const babelConfig = {
   presets: ['@babel/preset-typescript'],
-  plugins: [unwrap],
```

Comment:
In the core, this plugin is not used:

https://github.com/mui-org/material-ui/blob/1b563a153f5b2736f79a3f45573e2e2b66ba0a9a/docs/scripts/formattedTSDemos.js#L26